src/gui/accountsettings.cpp

```diff
@@ -77,6 +77,13 @@ AccountSettings::AccountSettings(const AccountStatePtr &accountState, QWidget *p
 
     connect(FolderMan::instance(), &FolderMan::folderListChanged, _model, &FolderStatusModel::resetFolders);
     if (accountsState()->supportsSpaces()) {
+        for (auto folder : FolderMan::instance()->folders()) {
```

Comment:
Hmm if we are connected I think the lambda should have been executed.
So do we need the extra round of counting here?

Reply:
You're right... If only the lambda were correct ;-)